routes/photo.js

```diff
@@ -0,0 +1,16 @@
+import express from 'express';
+import photoController from '../core/controllers/PhotoController.js';
+
+let router = express.Router();
+
+router.get('/:user-id/', photoController.getPhotosByAuthor);
+router.get('/public/:user-id/', photoController.getPhotosPublicByAuthor);
+router.get('/like/:photo-id/', photoController.getLikesById);
+router.get('/count/:user-id/', photoController.getCountByAuthor);
```

Comment:
What is this route use for?

Reply:
router.get('/:user-id/', photoController.getPhotosByAuthor); Dùng để lấy danh sách photo (private và public) của 1 user bất kỳ theo id.
router.get('/public/:user-id/', photoController.getPhotosPublicByAuthor); Dùng để lấy danh sách photo (chỉ public) của 1 user bất kỳ theo id.
router.get('/like/:photo-id/', photoController.getLikesById); Dùng để hiển thị số like của 1 photo(icon trái tym).
router.get('/count/:user-id/', photoController.getCountByAuthor); Dùng để hiển thị tổng số lượng photo.